### DISCUSSION TITLE
Trying to package ROCm suite as a Flatpak

### DISCUSSION THREAD

**Discussion by RushingAlien** [ROCm/ROCm#1897] [Q&A] [answered]
Hello! I am trying to package rocm as a flatpak, but I am confused with rocm-llvm
<https://github.com/xuhuisheng/rocm-build/blob/master/11.rocm-llvm.sh>
this one(targets Ubuntu) has
`-DLLVM_ENABLE_PROJECT='compiler-rt;lld;clang'`
<https://github.com/archlinux/svntogit-community/blob/packages/rocm-llvm/trunk/PKGBUILD>
meanwhile this(targets Arch) one has
`-DLLVM_ENABLE_PROJECT='llvm;compiler-rt;lld;clang'`

why is that?

**Comment by scchan** [comment, marked as answer]
`clang` enables both `llvm` and `clang` so `llvm` is redundant here, see https://clang.llvm.org/get_started.html
